### PR TITLE
test: replace AWS CLI installation with CircleCI orb

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [X] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Replace AWS CLI installation with CircleCI orb

### CODE REVIEW

File: .circleci/config/jobs/@jobs.yml [hunk 1]
```diff
@@ -235,14 +235,17 @@ eks_integration_tests:
...
+        override-installed: true
     - run:
         name: Create temp dir for logs
         command: mkdir -p /tmp/logs/test/integration/eks
     - run:
         name: Integration tests EKS
         command: |
+          npm install &&
           export KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG=$(./scripts/circleci-jobs/setup-integration-tests.py)
-          .circleci/do-exclusively --branch staging --job ${CIRCLE_JOB} npm run test:integration:eks:yaml
```
Comment (dkontorovskyy) [inline .circleci/config/jobs/@jobs.yml:245]: Removed `do-exclusively` as we control flow with the branches, so no need to do that

Comment (ivanstanev) [reply .circleci/config/jobs/@jobs.yml:245]: How do we control it with the branches?

Comment (dkontorovskyy) [reply .circleci/config/jobs/@jobs.yml:245]: https://github.com/snyk/kubernetes-monitor/blob/staging/.circleci/config/%40config.yml#L71

Comment (ivanstanev) [reply .circleci/config/jobs/@jobs.yml:245]: https://snyk.slack.com/archives/C022DN6CMHS/p1627308110027800?thread_ts=1627307326.026800&cid=C022DN6CMHS

**Review by ivanstanev — APPROVED** [review]

**Pull request merged**